Pr for resonite mp3

AwesomeTornado opened the pull request:
<!-- This template is provided for your convenience: feel free to delete it from your PR -->

This mod converts MP3 files into WAV files on import, so that they can be imported as audio files instead of as videos.

- [x] The mod id matches the harmony id if used by the mod and starts with the same id as your author folder
- [x] All used links are valid
- [x] Your ResoniteMod.Version must match the version being added in the manifest and follow [Semantic Versioning](https://semver.org/)
- [x] Your AssemblyVersion should match the mod version
- [x] You have included an accurate `sha256` hash for each artifact
- [x] Do not remove old mods. Instead, use the `deprecated` flag
- [x] Follow the [Resonite Policies and Guidelines](https://resonite.com/policies/) and [Mod Submission Guidelines](https://github.com/resonite-modding-group/resonite-mod-manifest/wiki/Submission-Guidelines)

XDelta commented on manifest/com.__Choco__/ResoniteMP3/info.json:
```diff
@@ -0,0 +1,55 @@
+			"releaseUrl": "https://github.com/AwesomeTornado/ResoniteMP3/releases/tag/ResoniteMP3",
+			"artifacts": [
+				{
+					"url": "https://github.com/AwesomeTornado/ResoniteMP3/releases/download/ResoniteMP3/__Choco__.ResoniteMP3.dll",
```
I added some reference docs on using System.IO for temp files on https://github.com/AwesomeTornado/ResoniteMP3/issues/3, There are also some other mods you could check out for what they do for importing or converting assets.

AwesomeTornado commented:
File names are now randomized
Temp files are stored in system temp folder
Temp files are now deleted on app start
Removed redundant NAudio libraries
Moved library install location to /rml_libs
Fixed the tag name and release title
Updated the name of the dll file to ResoniteMP3.dll
Changed version to 2.0.0

Thanks so much for the feedback, let me know if there are still things that need to be changed.

XDelta commented on manifest/com.__Choco__/ResoniteMP3/info.json:
```diff
@@ -0,0 +1,35 @@
+	"category": "Asset Importing",
+	"sourceLocation": "https://github.com/AwesomeTornado/ResoniteMP3",
+	"versions": {
+		"1.0.0": {
```
Version number here doesn't match version in dll being added, change to `2.0.0`

XDelta commented on .gitignore:
```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
```
Avoid adding extra files, discard this change/delete this file

XDelta commented on manifest/com.__Choco__/ResoniteMP3/info.json:
```diff
@@ -0,0 +1,35 @@
+{
+	"name": "ResoniteMP3",
+	"id": "com.ResoniteMP3",
```
id here should be `com.__Choco__.ResoniteMP3`

XDelta reviewed (dismissed):
A few more minor things left. Otherwise looks good.
